tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "skipLibCheck": true,
     "resolveJsonModule": true,
     "noImplicitAny": true,
+    "useUnknownInCatchVariables": false,
```

Comment:
why do we need this?

Reply:
cause the typescript >4.4.2 upgrade caused the linting to fail. 

Previously it defaults to `any`, after upgrade it defaults to `unknown`, this caused many will throw error cause we cannot access the properties in the error object unless we type guard it or do a check.

[Example of the errors](https://github.com/rationally-app/mobile-application/actions/runs/4869650347/jobs/8684418391)

This is the fastest way to retain the "old" behaviour. We can remove this option (it defaults to true) when we've updated the types; just felt that it maybe out of scope for this.